### PR TITLE
feature/issue-852

### PR DESCRIPTION
**Pull request by kuboosoft**
We can run dnf using polkit...

### CODE REVIEW

File: build_host_setup_fedora.sh [hunk 1]
```diff
@@ -24,6 +24,7 @@ sudo dnf install -y \
     curl \
     pkgconfig \
     libicu-devel \
+    procps-ng \
```
Comment (forslund) [inline build_host_setup_fedora.sh]: Is this a required dependency? Seems like it's a tool for browsing the proc file system

**Comment by forslund** [comment]
There's a `sudo` at the end of the file as well upgrading the virtualenvironment, shouldn't this also be using policykit?

**Comment by ethanaward** [comment]
@kuboosoft Any update on this? You might need to rebase on dev to fix those failing tests.

**Comment by penrods** [comment]
The build_host_setup_fedora.sh no longer exists, closing this PR unmerged.

**Pull request closed without merging**